frontend/src/components/Login.jsx

```diff
@@ -42,7 +42,7 @@ class Login extends Component {
         <div id="creatorLogin">
           <h2>For Creators:</h2>
           <FacebookLogin
-            appId="507675923424391"
+            appId={process.env.FB_APP_ID}
```

Comment:
So, does it mean we use process.env file in the frontend when it is deployed?  How did you make heroku read .env file?

Reply:
Hi @StellarApp, by adding [`systemvars: true`](https://github.com/StellarApp/capstone-influencer-search/blob/9ca48a52b1ad2cb7ca81e6104d1c3878304e5ccc/webpack.config.js#L26) in our webpack-config.js file, it allowed webpack to see the environment variables we defined in Heroku through the `Config Vars`.